scipy/stats/_stats_py.py

```diff
@@ -7166,29 +7172,35 @@ def warn_masked(arg):
     return Power_divergenceResult(stat, pvalue)
 
 
-def chisquare(f_obs, f_exp=None, ddof=0, axis=0):
-    """Calculate a one-way chi-square test.
+def chisquare(f_obs, f_exp=None, ddof=0, axis=0, *, sum_check=True):
+    """Perform Pearson's chi-squared test.
 
-    The chi-square test tests the null hypothesis that the categorical data
-    has the given frequencies.
+    Pearson's chi-squared test [1]_ assesses the null hypothesis that the observed
+    frequencies (counts) are a sample from a multinomial distribution; that is,
+    they are the counts after *N* total observations from a categorical
```

Comment:
"after" here sounds strange to me
either
"after N total observations are taken from ..."
or "after" -> "conditional on"